lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -8,7 +8,7 @@ def define
 
       query = Calculators::BenefitCapCalculatorDataQuery.new
       rates = query.data.fetch('rates')
-      benefits = query.data.fetch('benefits')
+      benefits = query.data.fetch('benefits').with_indifferent_access
```

Comment:
Is the call to `#with_indifferent_access` important here? It'd be good to mention it in the commit note if it is.